model/src/main/java/com/redhat/ipaas/model/connection/Connection.java

```diff
@@ -49,7 +50,7 @@ default String getKind() {
 
     Optional<String> getConnectorId();
 
-    String getConfiguredProperties();
+    Map<String, String> getConfiguredProperties();
```

Comment:
Should the value really just be a string?  Isn't that a bit limiting given that we've support in the form config for numbers, booleans, etc?